geopandas/base.py

```diff
@@ -658,7 +658,7 @@ def explode(self):
         Returns
         ------
         A GeoSeries with a MultiIndex. The levels of the MultiIndex are the
-        original index and an integer.
+        original index and an integer that counts the number of single geometries within a multi-part geometry.
```

Comment:
Can you wrap this on a next line to keep a line length limit of 80 chars? (PEP8)